src/test/java/core/basesyntax/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,120 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.FruitsStorage;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitDaoImplTest {
+    private static final String BANANA = "banana";
+    private static final String APPLE = "apple";
+    private static final String BANANA_UPPERCASE = "BANANA";
+    private static final String APPLE_UPPERCASE = "APPLE";
+    private static final String NON_EXIST_FRUIT = "pineapple";
+    private static FruitDao fruitDao;
+
+    @BeforeClass
+    public static void initFruitDao() {
+        fruitDao = new FruitDaoImpl();
+    }
+
+    @Before
+    public void fillStorage() {
+        FruitsStorage.fruitsStorage.put("banana", 100);
+        FruitsStorage.fruitsStorage.put("apple", 100);
+    }
+
+    @Test
+    public void getData_ok() {
+        Map<String, Integer> fromStorageForTest = fruitDao.getData();
+        int bananaExpected = 100;
+        int appleExpected = 100;
+        int bananaActual = fromStorageForTest.get(BANANA);
+        int appleActual = fromStorageForTest.get(APPLE);
+
+        assertEquals("Bananas quantity should be " + bananaExpected
+                + ", but was: " + bananaActual,
+                bananaExpected, bananaActual);
+        assertEquals("Apples quantity should be " + appleExpected
+                        + ", but was: " + appleActual,
+                appleExpected, appleActual);
```

Comment:
here, messages in assertEquals are redundant. also, you can write the comparing values right in the assertEquals, that would be shorter.